### ISSUE TITLE
IOS universal link handling and mobile flow with libraries 

### ISSUE THREAD

**Issue by zachferland** [uport-project/uport-mobile-issues#27]
The mobile flows with the libraries on IOS are not being handled properly. Links open up id.uport.me instead of mobile app. Apple can't seem to resolve the apple-app-site-association file at https://id.uport.me/apple-app-site-association. More details in slack infra channel.

**Comment by joshuabell** [comment]
there’s a possibility the universal link issue on iOS is an iOS UX issue which we’ll have to write code around. Kinda surprising it doesn’t work how it sounds like it’s supposed to work and how android works though. 

Gonna investigate to make sure the current UX flow is expected given iOS universal links

**Comment by joshuabell** [comment]
@zachferland After further investigation I belive on iOS app links work the way they are suppose to but not the way we want them too. We'll need to write some code around the known problems with Universal Links.

I believe I was pulled on this to checkout the iOS part but for me to continue improving the Universal Link Experience I need the ability to deploy to the websites "demo.uport.space", "id.uport.me" and I need to know who owns development in those areas so I can reach out for input. The reason for this request is I need to investigate UX improvements on both those sites and may require code/functionality updates.

This site was the best I found on Universal Links for iOS with the section "Known Problems with Apple Universal Links" the most pertinant: https://www.appsflyer.com/resources/everything-marketer-needs-to-know-deep-linking/universal-links/

**Comment by joshuabell** [comment]
Note: I will reach out to Levi, Greg and Jonny regarding UX changes to demo.uport.space and id.uport.me.

@pelle @zachferland My strategy is to work on my tasks for this sprint and come back to make optimizations to this one since I was pulled onto this task tangentially. I see it blocks another issue though. Please let me know if you'd like me resourced on right away and I'll shift gears.

**Comment by joshuabell** [comment]
Note: to deploy to demo.uport.space or id.uport.me you you'll need aws-cli keys and then you just run existing npm scripts in the repo

**Comment by joshuabell** [comment]
Universal links started working! The dream is alive!

Why this may have happened:
We changed the mime type of the `apple-app-site-association`  file a few days ago based off intel from a terrible third party half working random website that checks universal link implementation health. That change can take 48 hours to propagate to apple servers so say the docs. We noticed about 2 days later our universal links start working as you'd expect them to.
